packages/contentful--app-scripts/utils/analytics.js

```diff
@@ -0,0 +1,33 @@
+const Analytics = require('analytics-node');
+
+// Public write key scoped to data source
+const SEGMENT_WRITE_KEY = 'IzCq3j4dQlTAgLdMykRW9oBHQKUy1xMm';
+
+/**
+ *
+ * @param {object} properties tracking properties
+ * @param {string} properties.command triggered command e.g create-app-definition, upload, etc.
+ * @param {boolean} properties.ci value if --ci flag has been set
+ * @returns
+ */
+function track(properties) {
+  if (process.env.DISABLE_ANALYTICS) {
+    return;
+  }
+
+  const client = new Analytics(SEGMENT_WRITE_KEY);
+
+  try {
+    client.track({
+      event: 'app-cli-app-scripts',
+      properties,
+      anonymousId: 'anonymous',
```

Comment:
I'm not entirely sure, but I think we should not define this id here. This way we'll have a single user for all created apps. We should rather generate unique id every time or remove this field(?):
https://segment.com/docs/connections/sources/catalog/libraries/server/node/#track

![image](https://user-images.githubusercontent.com/8289447/211282328-77973ae1-59a4-4d10-b1ce-8b4ecae6c8d4.png)

As I understood it attaches anonymous id automatically:
![image](https://user-images.githubusercontent.com/8289447/211282265-9cebd483-2fef-4906-b83d-0b2a872c6dd9.png)

Reply:
Interesting, I think we need to provide at least one of them. So I think I will just create a random ID every call.